### PR TITLE
SIG: upgrade `SIG-Parser` to `SIG-DDL` and update some content

### PR DESCRIPTION
**Pull request by zimulala**
Upgrade `SIG-Parser` to `SIG-DDL` and update the "member list" and "roles and organization management".

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @tangenta

File: special-interest-groups/sig-ddl/member-list.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Members and their contributions:
+
+| user                                      | num_merged_PRs | earliest_PR_merged_time | latest_PR_merged_time |
+|:-----------------------------------------:|:--------------:|:-----------------------:|:---------------------:|
+| [hey-kong](https://github.com/ciscoxll)   | 45             | 2018-03-13 02:30:31     | 2019-03-16 02:00:35   |
+| [spongedu](https://github.com/spongedu)   | 28             | 2018-11-26 12:50:20     | 2019-11-26 17:51:25   |
+| [leiysky](https://github.com/leiysky)     | 8              | 2019-07-30 11:07:34     | 2019-08-16 23:51:46   |
+
+## Technical Leads
```
Comment (gingerkidney) [inline special-interest-groups/sig-ddl/member-list.md]:
```suggestion
## Tech Leads
```

File: special-interest-groups/sig-ddl/member-list.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Lead:
```
Comment (gingerkidney) [inline special-interest-groups/sig-ddl/member-list.md]:
```suggestion
Leads:
```

File: special-interest-groups/sig-ddl/roles-and-organization-management.md [hunk 3]
```diff
@@ -13,22 +13,25 @@ the **Active Contributor** community role in the TiDB/parser repository.
 
 Requirements:
 
-* Have at least 8 parser-related PRs merged within one year.
-* Nominated by at least 2 Parser SIG Reviewers, Committer or Maintainer.
+* Have at least 8 PRs merged within one year.
+* Nominated by at least 2 DDL SIG Reviewers, Committer or Maintainer.
```
Comment (gingerkidney) [inline special-interest-groups/sig-ddl/roles-and-organization-management.md]:
```suggestion
* Nominated by at least 2 DDL SIG Reviewers or roles above.
```

**Comment by zimulala** [comment]
PTAL @gingerkidney

**Pull request merged**